letsencrypt-apache/letsencrypt_apache/augeas_configurator.py

```diff
@@ -30,7 +33,8 @@ def __init__(self, *args, **kwargs):
         # Set Augeas flags to not save backup (we do it ourselves)
         # Set Augeas to not load anything by default
         my_flags = augeas.Augeas.NONE | augeas.Augeas.NO_MODL_AUTOLOAD
-        self.aug = augeas.Augeas(flags=my_flags)
+        my_dir = os.path.dirname(constants.AUGEAS_HTTPD_LENS)
```

Comment:
It seems like Augeas really needs directory. https://pythonhosted.org/setuptools/pkg_resources.html#resource-extraction: "If the named resource is a directory, then all resources within that directory (including subdirectories) are also extracted". So, please move the lens to a subdirectory (`letsencrypt-apache/letsencrypt_apache/augeas_lenses/httpd.aug`), and use `AUGEAS_LENSES = pkg_resources.resource_filename('letsencrypt_apache', 'lenses')`